types/superagent-retry-delay/superagent-retry-delay-tests.ts

```diff
@@ -0,0 +1,21 @@
+import * as superagent from 'superagent';
+import wrapSuperagent = require('superagent-retry-delay');
```

Comment:
I'm not sure why

```ts
import * as wrapSuperagent from 'superagent-retry-delay';
```

doesn't work, but it complains that

```
This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export.
```

even though it's using `export =` (just like `@types/superagent`) 🤷